tests/acceptance/features/webUISharingAcceptSharesToRoot/acceptShares.feature

```diff
@@ -79,7 +79,7 @@ Feature: accept/decline shares coming from internal users
     And user "Brian" has been added to group "grp1"
     And user "Alice" has shared folder "/simple-folder" with group "grp1"
     And user "Alice" has shared folder "/testimage.jpg" with user "Brian"
-    When the user browses to the files page
+    When the user has browsed to the files page
```

Comment:
```suggestion
    And the user has browsed to the files page
```

If we use the "has done..." form then it fits as a `Given` step, so use `And`here. This scenario is a bit "naughty" because it tests 2 things - that the auto-accept works (the shares are immediately visible) and that shares appear on the shared-with-me page. So it had 2 `When`/`Then` sets of steps. Alternatively, you could revert this change.